cobra/cmd/license_gpl_3.go

```diff
@@ -18,23 +18,22 @@ package cmd
 func initGpl3() {
 	Licenses["gpl3"] = License{
 		Name:            "GNU General Public License 3.0",
-		PossibleMatches: []string{"gpl3", "gpl", "gnu gpl3", "gnu gpl"},
+		PossibleMatches: []string{"gpl3", "gplv3", "gpl", "gnu gpl3", "gnu gpl"},
 		Header: `
 This file is part of {{ .appName }}.
 
 {{ .appName }} is free software: you can redistribute it and/or modify
-it under the terms of the GNU Lesser General Public License as published by
+it under the terms of the GNU General Public License as published by
```

Comment:
This is not a spec. This is a suggestion. And a  terrible suggestion that creates a bazillion variants of the GPL notices... I can detect them alright in [ScanCode](https://github.com/nexB/scancode-toolkit)  anyway but it is best to avoid this variants IMHO.